src/Admin/RunningMateAdmin.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use AppBundle\Entity\ApplicationRequest\RunningMateRequest;
+use AppBundle\Entity\ApplicationRequest\Theme;
+use League\Flysystem\Filesystem;
+use Misd\PhoneNumberBundle\Form\Type\PhoneNumberType;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Route\RouteCollection;
+use Sonata\Form\Type\BooleanType;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\CountryType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+
+class RunningMateAdmin extends AbstractAdmin
+{
+    protected $datagridValues = [
+        '_sort_order' => 'ASC',
+        '_sort_by' => 'name',
+    ];
+    private $storage;
+    private $manager;
+
+    public function __construct($code, $class, $baseControllerName, Filesystem $storage, IsAdherentManager $manager)
+    {
+        parent::__construct($code, $class, $baseControllerName);
+        $this->storage = $storage;
```

Comment:
Instead of overide the constructor, you can use setter injection like : 
```
<call method="setStorage">
    <argument type="service" id="app.storage" />
</call>
```
or directly with the `@required` annotation.

Same for the manager.